stenotype/stenotype.cc

```diff
@@ -598,8 +598,8 @@ int Main(int argc, char** argv) {
       CHECK_SUCCESS(NegErrno(testimony_connect(&t, flag_testimony.c_str())));
       CHECK(flag_threads == testimony_conn(t)->fanout_size)
           << "--threads does not match testimony fanout size";
-      CHECK(testimony_conn(t)->block_size == flag_blocksize_kb)
-          << "Testimony does not supply blocks of size " << flag_blocksize_kb
+      CHECK(testimony_conn(t)->block_size == flag_blocksize_kb * 1024)
+          << "Testimony does not supply blocks of size " << flag_blocksize_kb * 1024
```

Comment:
line 602 shouldn't change, since line 603 appends "KB" to this.  Line 601 definitely should change like you wanted it to, though.